src/support/topological_orders.h

```diff
@@ -27,22 +27,24 @@
 #include <variant>
 #include <vector>
 
+#include "support/index.h"
+
 namespace wasm {
 
 namespace TopologicalSort {
```

Comment:
Maybe worth a comment that this uses `Index` because it is meant to sort wasm things. In theory if we want to sort something larger we'd need to template this over the type or something (and turn `TopologicalSort` into a struct and not a namespace).